release_notes/42-release-notes.adoc

```diff
@@ -0,0 +1,435 @@
+| GA
+| DEP
+| DEP
+
+| `Vulnerability Management Requester` permission
+| GA
+| DEP
+| DEP
+
+| `Vulnerability Report Creator` permission
+| GA
+| GA
+| DEP
+
+| `Vulnerability Reports` and `Policy` permission
+| DEP
+| REM
+| NA
+
+| `Role` resource
+| DEP
+| REM
+| NA
+
+| `Scope Manager` system role and permission set
+| DEP
+| REM
+| NA
+
+| `ROX_FORCE_LOCAL_IMAGE_SCANNING` environment variable
+| DEP
+| REM
+| NA
+
+| CIS Docker v1.2.0 Compliance Standard
+| NA
+| NA
+| DEP
+
+|====
+
+
+[id="deprecated-features_{context}"]
+=== Deprecated features
+
+The following section provides additional information about deprecated features listed in the preceding table.
+
+* Red Hat has deprecated the following commands for build-time network policy generation (Technology Preview) feature:
+** `roxctl generate netpol`, use the `roxctl netpol generate` command instead.
+** `roxctl connectivity-map`, use the `roxctl netpol connectivity map` command instead.
+* Red Hat will remove the CIS Docker v1.2.0 standard from {product-title-short} Compliance checks starting in {product-title-short} 4.4.
+* Starting with version 4.0, {product-title-short} does not evaluate the risk associated with service account permissions for deployments.
+
+[id="removed-features_{context}"]
+=== Removed features
+
+The following section provides additional information about removed features listed in the preceding table.
+
+[id="dark-mode_{context}"]
+==== Dark mode
+Dark mode in {product-title-managed-short} had rendering issues with buttons and images, leading to an inconsistent experience. Therefore, the dark mode and its toggle button has been removed from the navigation. (ROX-16515)
+
+[id="pdf-export_{context}"]
+==== PDF export
+Red Hat has removed the PDF export feature from the Vulnerability Management dashboard. Instead, you can use the updated vulnerability reporting feature in Vulnerability Management 2.0 (Technology Preview). (ROX-16394)
+
+:FeatureName: Vulnerability Management 2.0
+include::snippets/technology-preview.adoc[]
+
+[id="report-configuration-without-an-associated-collection"]
+==== Report configuration without an associated collection
+
+In version 4.0, {product-title-short} released the collections feature that replaced access scopes used in report configurations. {product-title-short} automatically created equivalent collections for access scopes used in existing report configurations and migrated report configurations to use newly-created collections. If the migration failed, the report configurations became non-functional, and {product-title-short} logged the error messages in Central logs and in the {product-title-short} web portal.
+{product-title-short} 4.2 deletes the report configurations that could not be migrated. To avoid losing any existing report configurations, create valid collections for reports that failed the migration and attach the collection to the report configuration before you upgrade to {product-title-short} 4.2.
```

Comment:
Looks like you have a break here, but it shows up all as one paragraph in the preview. Maybe insert an extra line space?